app/javascript/packs/sdk.js

```diff
@@ -31,6 +32,16 @@ const runSDK = ({ baseUrl, websiteToken }) => {
       IFrameHelper.events.toggleBubble(state);
     },
 
+    showWidget() {
+      document.querySelector('.woot--bubble-holder').style.display = 'block';
```

Comment:
Just have a check if the element exists. If they used hideMessageBubble before, then I guess the element would not be available.

Also, a create a function similar to the below one.

```js
const toggleBubbleVisibility = visibility => {
  // document..
  // window.$chatwoot.hideMessageBubble = visibility;
}
```

Reply:
Shouldn't the name be `toggleWidgetVisibility` ? 

Another question - What happens to new incoming messages when the Widget is hidden?

Reply:
Let us disable only the bubble for now. The PR also does that. You can still call window.$chatwoot.toggle() to show the widget.

Reply:
Resolved in https://github.com/chatwoot/chatwoot/pull/4223/commits/5f7fa536ab1edd2413bb10d54d0b32cdc61b90cf